avocado/utils/disk.py

```diff
@@ -87,3 +87,28 @@ def get_filesystem_type(mount_point='/'):
             _, fs_file, fs_vfstype, _, _, _ = mount_line.split()
             if fs_file == mount_point:
                 return fs_vfstype
+
+
+def is_linux_fs_type(device):
```

Comment:
Do you think this could be made in a more generic utility function?  I mean, if you're getting the partition type, it makes sense to have a:

```python
def get_partition_info(partition_device):
```

Of sorts.  Then users can do:

```
if get_partition_info('/dev/nvme0n1p1')['type'] == 83:
```

Or maybe even wrap that line in another utility function.

Reply:
One other point is if you're interested in the partition type, or in the filesystem that actually lies in that partition, as they can differ.

Reply:
@clebergnu i am interested on  filesystem that lies on partition will adopt your suggestion